tracim/tracim/lib/email_fetcher.py

```diff
@@ -199,6 +213,7 @@ def run(self) -> None:
 
     def stop(self) -> None:
         self._is_active = False
+        del self.lock
```

Comment:
You make this del for filelock specific reason ?

Reply:
Just to be sure lock is released, del is same as release(force = True) according to 
https://github.com/benediktschmitt/py-filelock/blob/master/filelock.py .

Reply:
I guess the lock is free when no "with lock.acquire" is done. So i guess it is not necessary to delete it when you stop the daemon.

Reply:
so add a comment for explaining this.